cherrypy/wsgiserver/ssl_builtin.py

```diff
@@ -37,23 +37,28 @@ class BuiltinSSLAdapter(wsgiserver.SSLAdapter):
     certificate_chain = None
     """The filename of the certificate chain file."""
 
+    ciphers = None
+    """The ciphers list of SSL"""
+
     """The ssl.SSLContext that will be used to wrap sockets where available
     (on Python > 2.7.9 / 3.3)
     """
     context = None
 
-    def __init__(self, certificate, private_key, certificate_chain=None):
+    def __init__(self, certificate, private_key, ciphers, certificate_chain=None):
```

Comment:
When changing function signature, make sure you find all its invocations. In this case, identifying **all of them** is way too hard. At least don't add new argument somewhere in the middle of existing ones. Adding it to the end would be safer.